src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2237,6 +2237,42 @@
      * The {@code default} keyword.  This element has no
      * children.
      *
+     * <p>For example:</p>
+     * <pre>
+     * case 1:
+     *   x=1;
+     *   break;
+     * default:
+     *   x=3;
```

Comment:
your example is not complete and it will also hard to understand for users
if you selected to use `switch` statement to demonstrate usage of `default` keyword, then please include initial `switch` statement

you can use smth similar to `TokenTypes#LITERAL_SWITCH` example, find it in TokenTypes class
```
switch(type)
{
 case 1:
  x = 1;
  break;
 default:
  x = 2;
}
```

------

or you can use this example with default method

```
public interface Test {
    default void defaultMethod() {
    }
}
```

-------

or annotation example
```
@interface Test {
 int version() default 1;
}
```
-------

Main point is that example should be easy to understand yet comprehensive for users
Please make sure that is your code is compilable

Reply:
@timurt Done taken switch case